scripts/build_scripts/build_zips.py

```diff
@@ -17,7 +17,7 @@
 
 
 Example usage:
-  python build_zips.py --platform=macos --targets=auth --targets=firestore
+  python build_zips.py --platform=macos --apis=auth --targets=firestore
```

Comment:
Technically it takes either targets or apis (I'm fine with this change, just providing info)